src/CommunityToolkit.Maui.Core/Essentials/AndroidStorageConstants.android.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace CommunityToolkit.Maui.Core.Essentials;
+
+static class AndroidStorageConstants
```

Comment:
> It lives in Core, it should be public

In this case,  I disagree. These are just helper constants that we're using internally to avoid hard-coded strings. There isn't any added value to developers here.

Reply:
I also agree with @brminnick, we should make it public in another PR